subsystems/motor/ServoSubsystem.java

```diff
@@ -0,0 +1,242 @@
+package org.usfirst.frc4904.standard.subsystems.motor;
+
+
+import org.usfirst.frc4904.standard.commands.Idle;
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class ServoSubsystem extends Subsystem {
+	protected final Servo[] servos;
+	protected boolean isInverted;
+	protected double lastPosition;
+	// Constants from wpilib's Servo.java.
```

Comment:
Is it possible to set `protected static final double kMaxServoAngle = wpilib.Servo.kMaxServoAngle`?
Also why does it start with `k`? Shouldn't final constants have names in all caps separated by underscores?